AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -767,7 +767,7 @@ - (NSMutableDictionary *)getReportingParametersForUpdatedRelease:(NSString *)upd
   }
 
   // Skip if downloaded release not installed yet.
-  if ([lastDownloadedReleaseHashes rangeOfString:currentInstalledReleaseHash].location == NSNotFound) {
+  if (![self isCurrentReleaseWasUpdated:lastDownloadedReleaseHashes]) {
```

Comment:
I'd like us moving away from `if(!...)` as it's easy to overlook the bang. cc @jaelim-ms

Reply:
Sure. Does this looks good to you?